test/java/seedu/addressbook/parser/NameIsSimilarTest.java

```diff
@@ -20,7 +20,8 @@ public void setup() throws IllegalValueException{
     public void null_returnsFalse(){
         assertFalse(name1.isSimilar(null));
     }
-        
+    
+    @Test
```

Comment:
This changes should be in the previous commit, look for how you can cancel a commit.